bin/make-snake

```diff
@@ -96,6 +96,17 @@ function colourEcho
     colour_echo "$1"
 }
 
+function ensureContentDir
+{
```

Comment:
Awesome spot! :+1: 

Without the `-p` it prints `mkdir: cannot create directory '/home/.../Snake-content': File exists`